.travis.yml

```diff
@@ -9,7 +9,8 @@ git:
 
 before_script:
   - travis_retry ./bin/docker-up-and-check.sh
-  - npm install
+  - npm install -g npm@latest
+  - npm ci
```

Comment:
Travis is running `npm install` before getting to this part - it's automatic for Node projects that don't have a `install` section - which means `package-lock` could have been changed before `npm ci` runs.
The alternative is to override the default by moving these two lines to `install` section.
https://docs.travis-ci.com/user/customizing-the-build/#Customizing-the-Installation-Step

Reply:
Just saw https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Using-a-specific-npm-version which recommends `npm install -g npm@latest` should be on `before_install` section.